webapp/src/main/webapp/resources/template.soy

```diff
@@ -347,4 +347,39 @@
   </table>
   <p>Please fix the problem of stop names for the corresponding trip(s)!</p>
   <br><br>
+{/template} 
+
+/**
+ * Renders a basic explanation of the stops too close notice.
+ * @param notices : List<[stopId1 : string, csvRowNumberStop1 : number, stopId2 : string, csvRowNumberStop2 : number, tripBufferMeters : number]>
+ */
+{template .stopsTooClose}
+{let $numNotices: length($notices)/}
+  <p class="warning">Warning - Stops too close!</p>
+  <p>Description: Two stops are too close with each other.</p>
+  <p><b>{$numNotices}</b> Stops too close found in:</p>
```

Comment:
Same as previous, I think `<p><b>{$numNotices}</b> found:</p>` is sufficient

Reply:
Sounds reasonable~ Will go to adjust them all.

Reply:
Fixed. Will fixed in the corresponding following PRs.

Reply:
All changes are in PR #46